.github/workflows/ci_action.yml

```diff
@@ -0,0 +1,31 @@
+name: build
+
+on:
+  push:
```

Comment:
this wont let us run it on external MRs.
you want it on push for master and on pull request (see [here](https://github.com/sentinel-hub/sentinelhub-py/blob/master/.github/workflows/ci_action.yml))